examples/plot_localizer_analysis.py

```diff
@@ -54,8 +52,11 @@
 # Design matrix
 ########################################
 
-paradigm = load_paradigm_from_csv_file(paradigm_file)['0']
-
+paradigm = DataFrame.from_csv(paradigm_file)
+onset = [float(x[0].split(' ')[2]) for x in paradigm.iterrows()]
+name = [x[0].split(' ')[1] for x in paradigm.iterrows()]
+n_conditions = len(np.unique(name))
+paradigm = DataFrame({'onset': onset, 'name': name})
```

Comment:
You can greatly simplify this. I don't know what the first column (with only zeros) is so you can chose a better name if you want:

```python
df = DataFrame.from_csv(paradigm_file, sep=' ', header=None, index_col=None)
df.columns = ['no_clue_what_so_ever', 'name', 'onset']
n_conditions = len(df.name.unique())
```